### DISCUSSION TITLE
Blocked script execution in because the document's frame is sandboxed and the 'allow-scripts' permission is not set.

### DISCUSSION THREAD

**Discussion by Onepunchman708** [cefsharp/CefSharp#4501] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

110.0.280

### Operating System

Windows 11

### Architecture

arm64

### .Net Version

4.7.2

### Implementation

WPF

### Reproduction Steps

This happens to every product I click on, the message says: Unfortunately, something went wrong. Refresh the page and start again.

![image](https://user-images.githubusercontent.com/132936865/236893553-2aa80a1c-8d8d-4d81-88eb-2652fe25e7ec.png)

Here the URL :
https://www.amazon.nl/Revlon-Super-Lustrous-Lipstick-Raisin/dp/B000V5OQNG/ref=sr_1_14?pd_rd_r=f1785a04-25a3-4252-816c-300d8644e970&pd_rd_w=Uqnuj&pd_rd_wg=78xS3&pf_rd_p=72fd8ca0-c188-46d9-ac2b-6e1fbe4b5544&pf_rd_r=5W5JFPGZ88F704JT5P4Q&qid=1682580832&refinements=p_36%3A20314668031&rnid=16241722031&s=beauty&sr=1-14](https://nam12.safelinks.protection.outlook.com/?url=https:%2f%2fwww.amazon.nl%2fRevlon-Super-Lustrous-Lipstick-Raisin%2fdp%2fB000V5OQNG%2fref%3Dsr_1_14?pd_rd_r%3Df1785a04-25a3-4252-816c-300d8644e970%26pd_rd_w%3DUqnuj%26pd_rd_wg%3D78xS3%26pf_rd_p%3D72fd8ca0-c188-46d9-ac2b-6e1fbe4b5544%26pf_rd_r%3D5W5JFPGZ88F704JT5P4Q%26qid%3D1682580832%26refinements%3Dp_36%253A20314668031%26rnid%3D16241722031%26s%3Dbeauty%26sr%3D1-14&data=05%7c01%7cli.li%40centific.com%7c30471fc0b0d643ae376808db46f3f7ff%7c9b415834803a4da0afdcfe6b1d52d649%7c0%7c0%7c638181785928124123%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=Jmt7qw8w8zCukMAMbbt0rMYPYRXgZVgYJyswMUcOSOU%3D&reserved=0)

### Expected behavior

Open it correctly

### Actual behavior

There was a error message says: Unfortunately, something went wrong. Refresh the page and start again.

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
> Please only open an issue if you can reproduce the problem with version 112.3.0 or later.

Please don't open issues for old unsupported versions.

> Does this problem also occur in the CEF Sample Application
> Yes using WPF/OffScreen command line args

I'd suggest you upgrade to the current supported version. If the problem persists and the problem reproduces with the **CEF** sample application then you'll need to raise an issue at https://github.com/chromiumembedded/cef
